src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/TestGenieToolWindowState.kt

```diff
@@ -6,45 +6,35 @@ package nl.tudelft.ewi.se.ciselab.testgenie.toolwindow
  */
 data class TestGenieToolWindowState
 constructor(
-    var searchBudget: Int = 60,
-    var localSearchBudgetType: LocalSearchBudgetType = LocalSearchBudgetType.TIME,
-    var localSearchBudgetValue: Int = 5,
     var stoppingCondition: StoppingCondition = StoppingCondition.MAXTIME,
+    var searchBudget: Int = 60,
     var initializationTimeout: Int = 120,
     var minimizationTimeout: Int = 60,
     var assertionTimeout: Int = 60,
     var junitCheckTimeout: Int = 60,
-    var population: Int = 50,
-    var populationLimit: PopulationLimit = PopulationLimit.INDIVIDUALS
+    var populationLimit: PopulationLimit = PopulationLimit.INDIVIDUALS,
+    var population: Int = 50
 ) {
     private object DefaultToolWindowState {
+        val stoppingCondition: StoppingCondition = StoppingCondition.MAXTIME
         const val searchBudget: Int = 60
-        const val localSearchBudgetValue: Int = 5
         const val initializationTimeout: Int = 120
         const val minimizationTimeout: Int = 60
         const val assertionTimeout: Int = 60
         const val junitCheckTimeout: Int = 60
-        const val population: Int = 50
-        val localSearchBudgetType: LocalSearchBudgetType = LocalSearchBudgetType.TIME
-        val stoppingCondition: StoppingCondition = StoppingCondition.MAXTIME
         val populationLimit: PopulationLimit = PopulationLimit.INDIVIDUALS
+        const val population: Int = 50
```

Comment:
Out of interesting, why did you move state values around? Does it matter? (I mean `population`)

Reply:
We moved the state values around because the interpretation of the value depends on the the type that was selected, therefore it makes sense to first select the type and then change the value. 

E.g.
PopulationLimit = Individuals 
Then the population value is 50 Individuals
OR
PopulationLimit = Statements
Then the population value is 50 Statements

I think it is more intuitive if the category is before the value. 

If this is what you asked.